src/extensions/ConflictFixer/index.ts

```diff
@@ -47,9 +50,69 @@ export default class ConflictFixer implements Partial<Extension> {
         this.counter++;
 
         if (this.counter > 10) { // TODO: make configurable
+
             const msg = `[ShortNight]: Too many times(${this.counter}) of try fix conflict.`;
 
             if (DEBUG) {
+
+                /**
+                 * Holds the filter data from the event which have generated the conflicts
+                 * */
+                const filterData = ((ud) => {
+
+                    const eventsData = ud.map(ud => {
+                        return this.conflicts.event_body.map(eb => {
+                            if (eb.self.drawInfo.date === ud.date) {
+                                return JSON.stringify(ud);
+                            }
+                        })
+                    });
+
+                    const flatenTheEvents = eventsData.flat(1);
+
+                    return flatenTheEvents.filter(Boolean) as string[];
+                    
+                })(this.ext.components.timeline[0].drawInfo.events);
+
+                let errorMsg = `[ShortNight]:`;
+
+                if (filterData.length === 1) {
+
+                    const description = JSON.parse(filterData.find(Boolean) as string).description;
+
+                    errorMsg = `${errorMsg} Description “${description}” is overlapping the other event data.`;
+
+                } else {
+
+                    /**
+                     * Finding the difference between two date and return the day
+                     * */
+                    const dayDifference = ((dateList) => {
+
+                        const oneDayMs = 1000 * 60 * 60 * 24;
+                        
+                        const countDiff = (date1: Date, date2: Date) => Math.abs(date1.getTime() - (date2 || new Date(date2)).getTime()) / oneDayMs;
+    
+                        const dateListMS = dateList.map((v, i, arr) => countDiff(v, arr[i + 1]));
+    
+                        return dateListMS.filter(v => !Number.isNaN(v)).sort().reverse()[0];
+    
+                    })(filterData.map(v => JSON.parse(v)).map(v => new Date(v.date)));
+
+                    if (!dayDifference) {
+                        errorMsg = `${errorMsg} “Duplicate entry found for event.date”`;
+                    } else {
+                        errorMsg = `${errorMsg} “Too close entry found for event.date”`;
+                    }
+                }
+
+                if (filterData.length > 1) {
+
+                    errorMsg = `${errorMsg} ${filterData.toString()}`;
```

Comment:
Do you think that users would understand what the problem is when they saw this message?